plugins/client-resources/src/connection.ts

```diff
@@ -251,20 +255,31 @@ class Connection implements ClientConnection {
         this.delay = 3
         return
       }
-      if (resp.result === 'hello') {
```

Comment:
i think better to add a new filed to Hello response message with version, it will not break current users in this case.

Reply:
Done